src/foam/nanos/auth/User.js

```diff
@@ -303,26 +260,18 @@ foam.CLASS({
       documentation: `The date and time that the current password of the User
         will expire.`,
     },
-    // TODO: startDate, endDate,
```

Comment:
Are both these TODOs actually addressed?

Reply:
No. But there is nothing in the works for them. We have many ideas but not enough time to do them all.

If this becomes a requirement then we can do it.

This just seems like a 1 yr old sticky note.

Reply:
1 year old sticky notes can still be important.  It's not a question of hard requirements but more just documented alternative design ideas that haven't been resolved.  I've found 3-4 year old TODOs that predicted the very bug that I was investigating.  I see no reason to clean them up

Reply:
interesting. ok. reversed delete